src/cljs/coffee_re_frame/effects.cljs

```diff
@@ -37,3 +37,31 @@
   (.vibrate js/navigator duration))
 
 (re-frame/reg-fx :vibrate vibrate-effect)
+
+(def current-lock (atom nil))
+
+(defn lock-brightness []
+  (->
+   (.request (.-wakeLock js/navigator) "screen")
+   (.then (fn [new-lock]
+            (js/console.log "Locking screen brightness: " new-lock)
+            (reset! current-lock new-lock)))
+   (.catch (fn [error]
+             (js/console.error "Error locking screen brightness"
+                               error)))))
```

Comment:
Useful for dev work, but honestly if we don't have reporting, we could just throw this into a black hole